docs/workflow/testing/libraries/testing.md

```diff
@@ -15,6 +15,11 @@ build.cmd/sh -subset libs.tests
 build.cmd/sh -subset libs.tests -test -c Release
 ```
 
+- The following builds all tests for x86 architecture (**for x86 it's mandatory to build clr** even if you want to run libs tests only):
```

Comment:
This statement applies for all architectures, is it worth adding an entry here? This might confuse people where they already built coreclr for x86, i.e running: `build.cmd clr+libs -rc Release -arch x86` that will build clr on release and libraries on Debug for x86.

Reply:
>  is it worth adding an entry here

Do you mean that building clr is mandatory to run libs tests on other architectures as well?

Since you are the expert here, you can just provide a GH suggestion and merge it to my branch and then just squash the PR.

Reply:
note that you can also use mono instead of clr, the lines just need a runtime it doesn't matter which one

Reply:
```suggestion
- The following builds all tests for x86 architecture (**for any architecture it's mandatory to build a runtime (clr or mono)** even if you want to run libs tests only):
```